bitnami/mariadb/templates/NOTES.txt

```diff
@@ -26,9 +26,9 @@ Tip:
 
 Services:
 
-  echo Primary: {{ include "mariadb.primary.fullname" . }}.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain }}:{{ .Values.primary.service.port }}
+  echo Primary: {{ include "mariadb.primary.fullname" . }}.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain }}:{{ .Values.primary.service.ports.mysql }}
 {{- if eq .Values.architecture "replication" }}
-  echo Secondary: {{ include "mariadb.secondary.fullname" . }}.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain }}:{{ .Values.secondary.service.port }}
+  echo Secondary: {{ include "mariadb.secondary.fullname" . }}.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain }}:{{ .Values.secondary.service.ports.mysql }}
```

Comment:
I'm strongly against this change. It breaks backwards compatibility and there's no need for it all.